shotover/src/codec/cassandra.rs

```diff
@@ -773,6 +774,16 @@ impl Encoder<Messages> for CassandraEncoder {
 }
 
 impl CassandraEncoder {
+    pub fn set_startup_state_ext(&mut self, compression: String, version: Version) {
```

Comment:
what does `ext` in the function name here mean?

Reply:
external

Reply:
I was thinking extension. Maybe we should write it out fully as `set_startup_state_ext`